Event/include/Event/Version.h

```diff
@@ -0,0 +1,26 @@
+/**
```

Comment:
This file should not be committed.  Please remove it.

Reply:
Alright. I've removed it and committed the change.